build_projects/shared-build-targets-utils/Utils/HostVersion.cs

```diff
@@ -62,7 +62,7 @@ public override string ToString()
         //
 
         // Full versions and package information.
-        public static bool EnsureStableVersion => true;
+        public static bool EnsureStableVersion => false;
         public string LatestHostPrerelease => "servicing";
         public string LatestHostBuildMajor => CommitCountString;
         public string LatestHostBuildMinor => "00";
```

Comment:
You need to bump the 6 to 7 below to fix your CI issues.

Reply:
Duh, missed that one. Thanks for taking a look.